_includes/meetup2019032.html

```diff
@@ -0,0 +1,61 @@
+{% include components/meetuptitle.html number=31 title="Kubernetes around the globe" date="10/10/2019" host="CriticalTechWorks"%}
+
+<div class="columns content">
+  <div class="column is-one-third">
+    <div class="box">
+      <h3>10/10/2019</h3>
+      <h3>Agenda:</h3>
+      <ul>
+        <li>Welcome</li>
+        <li>Talk:
+          <strong>K8s in AWS: Europe, US and The Great Wall of China  </strong>
+        </li>
+        <li>Coffee/Networking break</li>
+        <li>Wrap up</li>
+      </ul>
+      <h3>Host</h3>
+      <figure class="image ">
+        <a href="https://www.criticaltechworks.com/" target="_blank"><img alt="CriticalTechWorks" class="width-is-128" src="assets/images/hosts/criticaltechworks.png" /></a>
+      </figure>
+    </div>
+  </div>
+  <div class="column is-auto">
+
+    <div class="box">
+      <article class="media">
+        {% include components/speakerimage.html imagename="m032_joaopeixoto.jpeg" %}
+
+        <div class="media-content">
+          <div class="content">
+            <h3>K8s in AWS: Europe, US and The Great Wall </h3>
+            {% include components/speakerhandle.html name="João Peixoto" linkedin="peixotojoao" %}
+            <p>
+              For the last year and a half, our invited speaker João Peixoto, have been working in DevOps teams 
+              developing solutions to be instantiated in Kubernetes cluster on AWS in there different regions: 
+              Frankfurt, North Virginia and Beijing. He's going to talk about the struggle to instantiate an 
+              enterprise grade Kubernetes cluster in AWS and all the tools that enable his team to do DevOps 
+              in a production environment.
+            </p>
+
+            <h4>About João Peixoto:</h4>
+            <p>
+              João is a person that was always passionate about software development and he really enjoy to 
+              work on new technologies. In fifteen years of professional experience he was fortunate to 
+              work on some of the top Portuguese companies in three different business areas 
+              (software development, telecommunications, power distribution) which allowed him to acquire a 
+              very rich and diverse set of technical skills and be familiar with different software development 
+              processes.
+            </p>
+
+            <!-- <h4>Media</h4>
+            <p>
+              <a href="#" target="_blank">Slide deck @ SpeakerDeck</a>
```

Comment:
Altera para os slides que agora estão aqui https://speakerdeck.com/devopsporto/k8s-in-aws-europe-us-and-the-great-wall-of-china